bridge_adaptivity/module/models.py

```diff
@@ -301,6 +309,15 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('module:group-detail', kwargs={'group_slug': self.slug})
 
+    def has_linked_sequences(self):
+        """Indicate that collection group has linked sequences."""
+        # sequence has link to group
+        return self.sequence_set.exists()
+
+    def has_linked_active_sequences(self):
+        """Indicate that collection group has linked not finished sequences."""
+        return self.sequence_set.filter(completed=False)
```

Comment:
ditto

BTW these methods are the same as in L203-209, it seems mixin class would be a good approach.

Reply:
actually it will not be Mixin, it will be base class.